jdk-1.7-parent/annotation/src/main/java/org/wicketstuff/annotation/scan/AnnotatedMountScanner.java

```diff
@@ -97,46 +98,16 @@
 {
 
 	/**
-	 * Get the Spring search pattern given a package name or part of a package name
-	 * 
-	 * @param packageName
-	 *            a package name
-	 * @return a Spring search pattern for the given package
-	 */
-	public String getPatternForPackage(String packageName)
-	{
-		if (packageName == null)
-			packageName = "";
-		packageName = packageName.replace('.', '/');
-		if (!packageName.endsWith("/"))
-		{
-			packageName += '/';
-		}
-
-		return "classpath*:" + packageName + "**/*.class";
-	}
-
-
-	/**
-	 * Scan given a package name or part of a package name and return list of classes with MountPath
-	 * annotation.
-	 * 
-	 * @return A List of classes annotated with &#64;MountPath
-	 */
-	public List<Class<?>> getPackageMatches(String pattern)
-	{
-		return getPatternMatches(getPatternForPackage(pattern));
-	}
-
-	/**
-	 * Scan given a Spring search pattern and return list of classes with MountPath annotation.
+	 * Scan a list of classes which are annotated with MountPath
 	 * 
-	 * @return A List of classes annotated with &#64;MountPath
+	 * @param mounts
```

Comment:
The parameter name is actually `patterns`.
Please give an example what a `pattern` could look like, so it is easier for beginners what to provide.